common/lib/xmodule/xmodule/modulestore/tests/factories.py

```diff
@@ -412,6 +413,7 @@ def _create(cls, target_class, **kwargs):  # lint-amnesty, pylint: disable=argum
                 module.submission_start = submission_start
             if submission_end:
                 module.submission_end = submission_end
+            store.update_item(module, user_id)
```

Comment:
This lets us set `has_score` for xblocks where it's a proper Field and for xblocks where it's just a class attribute. `update_item` will save changes to Fields back to the store.